test/kubemark/resources/hollow-node_template.yaml

```diff
@@ -55,7 +55,7 @@ spec:
           "--morph=kubelet",
           "--name=$(NODE_NAME)",
           "--kubeconfig=/kubeconfig/kubelet.kubeconfig",
-          "$(CONTENT_TYPE)",
```

Comment:
Let's remove it and remove it from the configmap as well. I'd like to go into a direction where the config is simple and only sets the things that are needed and differ from the defaults. The `kube-api-content-type` defaults to protobuf and I don't see a need for setting it differently in our tests. 

One can argue that it's nice to have a way to customize things, but for me it's [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it)